linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/correct_stig.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# make existing entries pass
+for acct in $(grep -ve ':\(!!\|\*\):' /etc/shadow | awk -F: '{print $1}'); do
+    chage -M 60 -d $(date +%Y-%m-%d) $acct
+done
+# add a guaranteed conforming entry
+echo 'max-test-user:$1$q.YkdxU1$ADmXcU4xwPrM.Pc.dclK81:18648:1:60::::' >> /etc/shadow
```

Comment:
This user has minimal password age set to 1, but the default value of var_accounts_minimum_age_login_defs.var is 7, so it should be an incorrect entry, but the comment above says that this command "adds a guaranteed conforming entry". If I understand it well, the minimal age isn't tested by this rule, but it's mind-boggling. I propose either adding an explanatory comment here or adding an user which conforms to both requirements at the same time.

Reply:
I've changed the default value of the variables involved in the check to align with STIG seletion in 3036f81148f9691a226b2314739af7971f406969